<launch>
    <machine name="local" address="localhost" default="true" />

      <arg name="node_start_delay" default="5.0" />
      <arg name="input_file" default="/home/dmo/Documents/diplom/dumps/compressed_dump_8.txt" />
      <arg name="output_file" default="/home/dmo/Documents/diplom/dumps/shifted_map_8.txt" />
      <arg name="shift_x" default="5" />
      <arg name="shift_y" default="0" />

      <node name="rviz_first_map" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find map_merging)/rviz/debug.rviz" />

      <node name="rviz_second_map" pkg="rviz" type="rviz" respawn="true"
            args="-d $(find map_merging)/rviz/debug_second_map.rviz" />
     
      <node pkg="map_merging" type="map_translation" name="map_translation" output="screen"
          launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' ">
            <param name="input_file" value="$(arg input_file)"/>
            <param name="output_file" value="$(arg output_file)"/>
            <param name="shift_x" value="$(arg shift_x)"/>
            <param name="shift_y" value="$(arg shift_y)"/>
        </node>
</launch>